src/server/src/test/resources/cassandra-reaper-at.yaml

```diff
@@ -10,6 +10,7 @@ storageType: memory
 incrementalRepair: false
 enableDynamicSeedList: false
 jmxConnectionTimeoutInSeconds: 300
+datacenterAvailability: LOCAL
```

Comment:
any reason to not use the default of `ALL` ?
(i would think it's only the cassandra backend that we want to test with `LOCAL`)

Reply:
it won't work if 2 nodes aren't reachable through JMX, which is the case in CircleCI tests

Reply:
yeah, ofc.